src/Schema/Events/System.ts

```diff
@@ -27,3 +27,31 @@ export interface TimeOnPage {
   context_page_owner_id?: string
   context_page_owner_slug?: string
 }
+
+/**
+ * A user loads a sale screen on the iOS app.
+ *
+ * This schema describes events sent to Segment from [[saleScreenLoadComplete]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "saleScreenLoadComplete",
+ *    context_page_owner_type: "auction",
+ *    context_page_owner_id: "5f841d4044f91e000fd0acc4",
+ *    context_page_owner_slug: "finarte-modern-and-contemporary-art-8",
+ *    jump_to_lai_interface: true,
+ *    load_time_ms: 2000,
+ *    number_of_lots: 276
+ *  }
+ * ```
+ */
+export interface SaleScreenLoadComplete {
+  action: ActionType.saleScreenLoadComplete
+  context_page_owner_type: PageOwnerType
```

Comment:
If this is iOs specific we should use `context_screen_owner_`... and `ScreenOwnerType`.

Reply:
ah good catch! i will update